ingestion/functions/parsing/ch_zurich/zurich_test.py

```diff
@@ -0,0 +1,57 @@
+import csv
+import json
+import os
+import tempfile
+import unittest
+from datetime import date
+
+import pytest
+
+from ch_zurich import zurich
+
+_SOURCE_ID = "abc123"
+_SOURCE_URL = "https://gd.zh.ch/internet/gesundheitsdirektion/de/themen/coronavirus.html"
+_PARSED_CASE = (
+    {
+        "caseReference": {
+            "sourceId": _SOURCE_ID,
+            "sourceUrl": _SOURCE_URL
+        },
+        "location": {
+            "country": "Switzerland",
+            "administrativeAreaLevel1": "Zurich",
+            "geoResolution": "Admin1",
+            "name": "Zurich canton",
+            "geometry": {
+                "longitude": "8.651071",
+                "latitude": "47.42568"
+            }
+        },
+        "events": [
+            {
+                "name": "confirmed",
+                "dateRange":
+                        {
+                            "start": "02/27/2020Z",
+                            "end": "02/27/2020Z"
+                        }
+            }
+        ],
+        "demographics": {
+            "ageRange": {
+                "start": 33,
+                "end": 33
+            },
+            "gender": "Female"
+        },
+    })
+
+
+class CHZurichTest(unittest.TestCase):
```

Comment:
Does this play nice for runs of `pytest` at the top-level `ingestion/functions` dir? Is there a display/functional difference between this and vanilla test files as for india?

Reply:
that's just so I can use self.assertCountEqual which shows much better output than a pure assert.
Those tests are still discovered by pytest, I just run it the same way as I do for the others:

```
python3.8 -m pytest .
============================================================= test session starts =============================================================
platform darwin -- Python 3.8.5, pytest-6.0.1, py-1.9.0, pluggy-0.13.1
rootdir: /Users/timothe/healthmap/healthmap-gdo-temp/ingestion/functions, configfile: pytest.ini
plugins: requests-mock-1.8.0
collected 19 items

parsing/ch_zurich/zurich_test.py .                                                                                                      [  5%]
parsing/common/python/parsing_lib/parsing_lib_test.py .......                                                                           [ 42%]
parsing/india/india_test.py .                                                                                                           [ 47%]
retrieval/retrieval_test.py ..........                                                                                                  [100%]

============================================================= 19 passed in 4.76s ==============================================================
```

Reply:
Got it, thanks!